service-base/src/main/java/fi/nls/oskari/domain/map/userlayer/UserLayerStyle.java

```diff
@@ -47,6 +49,35 @@ public void populateFromJSON(final JSONObject stylejs) throws JSONException {
 
     }
 
+    public JSONObject parseUserLayerStyle2JSON(){
```

Comment:
Maybe we should consider using Jackson and Json annotations for this. This quickly gets very complicated when the POJO model corresponds to the  doesn't correspond to the JSON model.

Reply:
Anyway we should probably add a unit test that creates a UserLayerStyle object by hand, sets some value to all the fields and then calls `parseUserLayerStyle2JSON()` and `populateFromJSON(JSONObject json)` and checks that the all the fields match the original UserLayerStyle.